src/components/Export.tsx

```diff
@@ -289,4 +294,7 @@ export const ExportDialog: ExportDialogComponent = ({
   );
 };
 
+export const render = (props: ExportDialogProps) =>
+  renderOverlay({ Overlay: ExportDialog, props });
+
 export default ExportDialog;
```

Comment:
In the future, would probably make sense to rename this file to `ExportDialog`. Didn't do it in this PR to limit confusion